webapp-django/crashstats/crashstats/static/crashstats/css/screen.less

```diff
@@ -430,17 +430,9 @@ html.production {
             }
             th.sortable {
                 padding-right: 17px;
-                background: url(../../img/2.0/sort-none.png) no-repeat right center;
             }
             th.sortable:hover {
                 cursor: pointer;
-                background-image: url(../../img/2.0/sort-none-hover.png);
-            }
-            th.sortable.sort-desc {
-                background-image: url(../../img/2.0/sort-desc.png);
-            }
-            th.sortable.sort-asc {
-                background-image: url(../../img/2.0/sort-asc.png);
```

Comment:
Can you explain this change? Why do you have to change the css? Plus it seems to me that you are removing "features". Unless those rules are never used?